base.css

```diff
@@ -1,10 +1,89 @@
-@import "intro.css";
-@import "index.css";
-
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
   font-size: 100%;
```

Comment:
In the prior course font size was set to 100% as a reference. 
So whenever a change was made in any element it has a reference to the global 100%. It was a cheap Udemy course! ;)

Reply:
As i see right now, do you reviewed an old version of the base.css? Have to check since we made some changes to the commits.

Font size correctes (removed)